addons/udes_stock/models/stock_picking.py

```diff
@@ -636,6 +636,7 @@ def _prepare_move(self, pickings, products_info, **kwargs):
                 product = product_info.get("product")
                 uom_qty = product_info.get("uom_qty") or product_info.get("qty")
                 uom_id = product_info.get("uom_id") or product.uom_id.id
+                expected_date = product_info.get("expected_date")
```

Comment:
I would move this in its own commit, so we have the tests in its own commit and changes on two different modules in separate commits.